gdal/ogr/ogrsf_frmts/ngw/drv_ngw_raster.html

```diff
@@ -0,0 +1,164 @@
+<html>
+<head>
+    <title>NextGIS Web</title>
+</head>
+
+<body bgcolor="#ffffff">
+
+<h1>NextGIS Web</h1>
+
+NextGIS Web - is a server GIS, which allows to store and edit geodata and to
+display maps in web browser. Also NextGIS Web can share geodata with other
+NextGIS software.
+<p>
+
+NextGIS Web has the following features:
+
+<ul>
+    <li> Display maps in a web browser (different maps with different layers and
+        styles)
+    <li> Flexible permissions management
+    <li> Load geodata from PostGIS or import from GIS formats (ESRI Shape,
+        GeoJSON or GeoTIFF)
+    <li> Load vector geodata in the following formats: GeoJSON, CSV, ESRI Shape
+    <li> Import map styles from QGIS project or set them manually
+    <li> Act as a server for TMS, WMS, WFS
+    <li> Act as a client for WMS
+    <li> User can add photos to records, change record attributes via web
+        interface or WFS-T protocol
+</ul>
+
+NextGIS Web - is an open source software (license GPL v2+, see
+<a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html">
+GNU General Public License, version 2</a>).
+
+<h2>Driver</h2>
+
+NextGIS  Web supports several raster types:
+
+<ul>
+    <li>Raster style
+    <li>Vector style
+    <li>WMS layer
+    <li>WMS Service
+    <li>Web map as combination of raster and vector styles
+</ul>
+</p>
+Each NextGIS Web raster layer can have one or more raster styles.<br>
+Each NextGIS Web vector or PostGIS layer can have one or more vector style (QGIS
+qml or MapServer xml).<br>
+WMS layer is one or more layers from external WMS service. This layer has no
+styles.<br>
+WMS Service is usual WMS protocol implementation.
+<p>
+Now NGW driver supports only raster and vector styles and WMS layers. You can get
+raster data as tiles or image (only tiles are supported now).
+</p>
+The driver supports read-only operations on rasters.
+
+<h2>Dataset name syntax</h2>
+
+The minimal syntax to open a NGW datasource is:
+
+NGW:[NextGIS Web URL][/resource/][resource identifier]
+
+<ul>
+    <li><b>NextGIS Web URL</b> may be an url to nextgis.com cloud service (for
+        example, https://demo.nextgis.com), or some other url including port and
+        additional path (for example, http://192.168.1.1:8000/test).
+    <li><b>resource</b> is mandatory keyword dividing resource identifier from
+        the rest of URL.
+    <li><b>resource identifier</b> this is positive number from 0 and above. This
+        may be an resource group, vector, PostGIS or raster layer, style.
+</ul>
+
+If identifier is resource group, all vector layers, PostGIS, raster layers, styles
+will listed as child resources. In other case this will be a separate raster.
```

Comment:
will be

Reply:
Fixed per 4579aeab927d70ddab6142340bf5550c6789b42e